challenges/classes.js

```diff
@@ -1,7 +1,33 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker2 {
+
+    constructor(dimensions) {
+
+  this.length = dimensions.length;
+  this.width = dimensions.width;
+  this.height = dimensions.height;
+
+}
+  volume (width, height) {
+  return length * width * height;
+  }
+
```

Comment:
you had this area correct as we had our 1 on 1, can you push up the changes so i can merge. you had these without the parameters and with the this.length type syntax in the return